ingestion/src/metadata/ingestion/source/database/clickhouse/queries.py

```diff
@@ -58,7 +58,7 @@
 	name as view_name,
 	database as schema,
 	create_table_query as view_def
-from system.tables where engine = 'View'
+from system.tables where engine in ['MaterializedView', 'View']
```

Comment:
also with this pr let's mark the materialised view as [this](https://github.com/open-metadata/OpenMetadata/blob/c101b3c9d195e342bc24f524aa728aeaaabf0d5e/openmetadata-spec/src/main/resources/json/schema/entity/data/table.json#L38) type